torch/csrc/api/include/torch/mps.h

```diff
@@ -5,6 +5,16 @@
 #include <cstddef>
 #include <cstdint>
 
+#ifdef __OBJC__
+#include <Foundation/Foundation.h>
+#include <Metal/Metal.h>
+typedef id<MTLCommandBuffer> MTLCommandBuffer_t;
+typedef dispatch_queue_t DispatchQueue_t;
+#else
+typedef void* MTLCommandBuffer_t;
+typedef void* DispatchQueue_t;
```

Comment:
Please use `using` instead of `typedefs` in C++ headers.
```suggestion
using  MTLCommandBuffer_t = id<MTLCommandBuffer>;
using DispatchQueue_t =  dispatch_queue_t;
#else
using  MTLCommandBuffer_t = void*;
using  DispatchQueue_t = void*;
```